bot/config.py

```diff
@@ -42,6 +42,8 @@
 
 line_img_url = "https://cdn.discordapp.com/attachments/581139962611892229/692712698487767080/animated_line.gif"
 
+WEATHER_API_KEY = "46b482193da9b57cf56fa98ff8855b55"
```

Comment:
This is incredibly bad, you've just exposed your API key, this needs to be a private `.env` variable and you'll need to reset your API key as you made it public in an open repository where everyone can see it and use it. Please remove this key, regenerate your own key and use an environment variable in `.env` for this (environmental variables can be loaded with `os.environ["name_of_env"]` or `os.getenv("name_of_env", default_value)`)